pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -80,8 +80,11 @@ const (
 	attrDestUser      = "destination.user"      // service account, e.g. "bookinfo-productpage".
 	attrConnSNI       = "connection.sni"        // server name indication, e.g. "www.example.com".
 
+	// attributes that could be used in a ServiceRole rules.
```

Comment:
What does this comment mean? User cannot use these pseudo attributes in ServiceRole. It's actually internal implementation detail about how do we generate them in Envoy config.

Reply:
Sorry it was confusing. Edited!